### PR TITLE
Create CODEOWNER

### PR DESCRIPTION
**Pull request by bmorelli25**
### Summary

This PR adds the @elastic/platform-docs team as a CODEOWNER to all files in this repository. An approving review from at least one member of the team will be required to merge all PRs.

### CODE REVIEW

File: .github/CODEOWNER [hunk 1]
```diff
@@ -0,0 +1 @@
+* @elastic/platform-docs
```
Comment (jrodewig) [inline .github/CODEOWNER:1]:
```suggestion
* @elastic/platform-docs

/docs/en/stack/ml @ml-docs
```

@szabosteve @chriscressman Let me know if there is a better Elastic team alias to use here.

**Review by jrodewig — DISMISSED** [review]
I think the filename should be `CODEOWNERS`.

**Review by chriscressman — APPROVED** [review]
LGTM

File: .github/CODEOWNER (hunk 1 above)
Comment (bmorelli25) [reply .github/CODEOWNER]: I went with @elastic/mlr-docs instead @elastic/ml-docs as `ml-docs` is a child of `mlr-docs`.

**Review by jrodewig — APPROVED** [review]

**Pull request merged**